test/unit/plugins/test_plugin_sort_dictionary.py

```diff
@@ -0,0 +1,13 @@
+from tracardi.process_engine.action.v1.sort_dictionary import SortedDictAction
+from tracardi.service.plugin.service.plugin_runner import run_plugin
+
+
+def test_sort_dictionary_plugin():
+    properties = {"data": {"first": 1, "second": 2, "fifth": 5, "third": 3}}
+    init = dict(
+        data='event@properties.data',
+        direction='asc',
+    )
+
+    result = run_plugin(SortedDictAction, init, properties)
+    assert result.output.value == {[["first", 1], ["second", 2], ["third", 3], ["fifth", 5]]}
```

Comment:
I see the changes in the plugin regarding the result. I believe that  after the changes this test will fail.

Reply:
Ahh. You're right. Thanks for spotting that